account-management-api/sonar-project.properties

```diff
@@ -1,6 +1,6 @@
-sonar.organization=alphagov
-sonar.projectKey=di-one-login-home-stubs-account-management-api
-sonar.projectName=di-one-login-home-stubs-account-management-api
+sonar.organization=govuk-one-login
+sonar.projectKey=account-management-stubs-api
```

Comment:
Nit picking, could this get confused with another teams project now we don't have anything in the name to indicate its a Home team project?  Also should the project key be the same as the github repo name?

Reply:
Hey Andrew, my thought was to align with the repo name. I did pass it by @alex9smith who was happy with the convention as per [thread](https://gds.slack.com/archives/C011Y5SAY3U/p1705417749817989?thread_ts=1705331996.296259&cid=C011Y5SAY3U). Although let me know if you have any other thoughts. The sonar.projectKey is the projects unique key which I've aligned with the four project configurations in sonar. There are four sonar configurations in this repo so I set up four projects, although it could be worth exploring consolidating that into one and perform single analysis across the repo which I wanted to flag with the team, although this may require some further restructuring.

Reply:
Ok, I understand the thinking now.  It makes sense and, given this is a test library, I don't think its worth spending more effort on the names of things.